Dockerfile

```
@@ -112,12 +112,12 @@ RUN set -eux \
 
 
 # zlib - http://zlib.net/
-ARG ZLIB_VERSION=1.2.13
+ARG ZLIB_VERSION=1.3
 
 RUN set -eux \
     && echo "Building zlib ${ZLIB_VERSION}..." \
     && cd /tmp \
-    && curl -LO "http://zlib.net/zlib-${ZLIB_VERSION}.tar.gz" \
+    && curl -LO "https://zlib.net/zlib-${ZLIB_VERSION}.tar.gz" \
```

Let's use this one to be consistent from now on. 

```suggestion
    && curl -LO "https://www.zlib.net/fossils/zlib-${ZLIB_VERSION}.tar.gz" \
```